packages/engine/README.md

```diff
@@ -365,7 +365,7 @@ The engine (and CLI) currently logs to both stderr, and to the `./log` directory
 
 ## Main Concepts
```

Comment:
This section is actually being superseded by #410 which should have been merged yesterday but I think a few other PRs jumped in front of it and stopped it being in-sync with main :(